pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -94,19 +119,11 @@ def __enter__(self):
         print(msg)
         bokchoy_utils.start_servers(self.default_store)
 
-        msg = colorize('green', "Waiting for servers to start...")
-        print(msg)
-        bokchoy_utils.wait_for_test_servers()
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        super(BokChoyTestSuite, self).__exit__(exc_type, exc_value, traceback)
-
-        msg = colorize('green', "Cleaning up databases...")
-        print(msg)
-
-        # Clean up data we created in the databases
-        sh("./manage.py lms --settings bok_choy flush --traceback --noinput")
-        bokchoy_utils.clear_mongo()
+    def interruptable_prompt(self):
+        """
+        Stops running process and waits for user input before continuing.
+        """
+        sh('read -p "Press enter to stop servers." PROMPTVAR')
```

Comment:
Oh, also people have had trouble in the past messing up their local devstack's .noseid files by hitting ^C multiple times quickly to kill running bok-choy tests. Although that falls into some "user error" gray area, we should try to make sure that issue doesn't become *more* prevalent. I'm not sure exactly how, but maybe by displaying text that instructs them to hit ^C once only?